tests/tasks/setup_ipa.yml

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
this file will conflict with #23 . Remove it from your commit and base your branch on top of #23.